### ISSUE TITLE
Possible security vulnerability

### ISSUE THREAD

**Issue by bwmills** [eugeneware/png-file-stream#6]
Hi @eugeneware,

Installing `npm` on macOS 10.14.1 shows a security vulnerability in a related dependency. 

The vulnerability is well known and has been fixed in [minimatch](https://nodesecurity.io/advisories/118) version 3.0.2 and later. The issue appears to be that version 1.0.0 of `png-file-stream` has these dependancies.

```
npm ls minimatch@'<3.0.4'
```
```
wizzy@0.6.0 /Users/.../.../wizzy
└─┬ png-file-stream@1.0.0
  └─┬ glob-stream@3.1.18
    ├─┬ glob@4.5.3
    │ └── minimatch@2.0.10
    └── minimatch@2.0.10
```

`npm audit` shows:

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ minimatch                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=3.0.2                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ png-file-stream                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ png-file-stream > glob-stream > glob > minimatch             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/118                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ minimatch                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=3.0.2                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ png-file-stream                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ png-file-stream > glob-stream > minimatch                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/118                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 2 high severity vulnerabilities in 849 scanned packages
  2 vulnerabilities require manual review. See the full report for details.
```

`npm audit fix` cannot resolve the issue per the `png-file-stream` 1.0.0 dependencies.

All other packages on my system can run `minimatch` 3.0.4 except these. I checked on `glob-stream` and `glob` and they both have newer versions that support the newer (fixed) `minimatch`.

It may well be that code in `png-file-stream` is not vulnerable per the original *Regular Expression Denial of Service* from the older versions of `minimatch` - just wanted to bring this to your attention.

**Comment by eugeneware** [comment]
Thanks @bwmills I've published 1.1.0 which is now passing audit free. Thanks for the heads up!

**Comment by bwmills** [comment]
Yep sure @eugeneware - many thanks for publishing the change!
